staging/script.xsqueeze/resources/settings.xml

```diff
@@ -45,4 +45,21 @@
        <setting label="(but if it is too low you may experience crashes crashes)" type="lsep"/>
        <setting id="waitOnAS" type="number" label="Max wait seconds for artist slideshow to exit cleanly... (default 10)" default="10"/>
     </category>
+    <category label="HDMI CEC">
+       <setting label="Please go to https://github.com/trainman419/python-cec and read about" type="lsep"/>
```

Comment:
it´s done :-)

Am 11.04.2015 um 03:06 schrieb bossanova808 <notifications@github.com>:

> In staging/script.xsqueeze/resources/settings.xml:
> 
> > @@ -45,4 +45,21 @@
> >         <setting label="(but if it is too low you may experience crashes crashes)" type="lsep"/>
> >         <setting id="waitOnAS" type="number" label="Max wait seconds for artist slideshow to exit cleanly... (default 10)" default="10"/>
> >      </category>
> > +    <category label="HDMI CEC">
> > +       <setting label="Please go to https://github.com/trainman419/python-cec and read about" type="lsep"/>
> One last thing - I would probably mention here it is included in OE, Raspbian etc, if there is room.
> (Already included with Openelec & Rapsbian) .... or similar
> 
> Should be good to merge then - thanks!
> 
> —
> Reply to this email directly or view it on GitHub.
>